std/encoding.d

```diff
@@ -281,6 +281,55 @@ import std.typecons;
         "\uFFFD","\uFFFD","\uFFFD","\uFFFD","\uFFFD","\uFFFD","\uFFFD","\uFFFD",
     ];
 
+    // HELPER FUNCTIONS
```

Comment:
Note, these were copied as-is to this unittest. I'm not in love with the implementation, but wanted to keep the PR simple.

Reply:
This movaroo wasn't quite needed because it doesn't import anything. But fine by me.